delfin/task_manager/scheduler/schedulers/telemetry/performance_collection_handler.py

```diff
@@ -74,7 +76,9 @@ def __call__(self):
             current_time = int(datetime.now().timestamp())
 
             # Times are epoch time in milliseconds
-            end_time = current_time * 1000
+            timestamp_offset = \
+                self.driver_api.get_timestamp_offset(self.ctx, self.storage_id)
```

Comment:
catch not implemented exception and default to 0

Reply:
Done.
1. catch NotImplementedError exception.
2. default to 0 if NotImplementedError catched.